### PR TITLE
chore(repo): Add GitHub Issue and PR templates

### PR DESCRIPTION
**Pull request by lusu007**

### CODE REVIEW

**Comment by lusu007** [comment]
I need some input regarding the checklist in the PR template @meyfa. What points should we add?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Bug report
+description: Create a bug report if you found a problem with foreman
```
Comment (meyfa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]:
```suggestion
description: Create a bug report if you found a problem with Foreman
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue. Before opening a bug report, please review our documentation.
+        If you cannot find a solution, kindly complete the following template with as much information as possible.
+
+  - type: input
+    attributes:
+      label: Version
+      description: Version of foreman you are using.
```
Comment (meyfa) [inline .github/ISSUE_TEMPLATE/bug_report.yml:13]:
```suggestion
      description: The version of Foreman you are using.
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+name: Feature request
+description: Suggest a new feature for foreman
```
Comment (meyfa) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]:
```suggestion
description: Suggest a new feature for Foreman
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  - type: textarea
+    attributes:
+      label: Desired feature
+      description: Provide a clear and concise description of the desired outcome.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Considered Alternatives
```
Comment (meyfa) [inline .github/ISSUE_TEMPLATE/feature_request.yml:22]:
```suggestion
      label: Considered alternatives
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      description: Describe any alternative solutions or features you've thought about.
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Additional Context
```
Comment (meyfa) [inline .github/ISSUE_TEMPLATE/feature_request.yml:29]:
```suggestion
      label: Additional context
```

**Review by meyfa — COMMENTED** [review]
LGTM besides some minor capitalization changes.

**Comment by meyfa** [comment]
> I need some input regarding the checklist in the PR template @meyfa. What points should we add?

Good question. The PR template is already asking for a proper description and issue reference, so that is out of the way. Depending on the change, we could ask for documentation, and/or tests — but this depends on the PR content, so it might be difficult to turn into an unambiguous rule.

Maybe we can go with just the Conventional Commit one for now and see what comes up in the future and add that.

**Review by meyfa — APPROVED** [review]

**Pull request merged**